src/app/vault/ciphers.component.ts

```diff
@@ -58,6 +63,7 @@ export class CiphersComponent extends BaseCiphersComponent implements OnDestroy
 
     async ngOnInit() {
         this.userHasPremiumAccess = await this.userService.canAccessPremium();
+        this.organizationNames = await this.loadOranizations();
```

Comment:
Typo in method name loadOr**g**anizations() and possibly rename to `loadOrganizationNames`